_partials/git.md

```diff
@@ -0,0 +1,47 @@
+## Git
+
+### Installation
+
+[`git`](https://git-scm.com/) is a command line software used for version control.
+
+To install `git`:
+- Open a terminal
+- Copy and paste the following commands:
+
+```bash
+sudo apt update
+sudo apt install -y git
+````
+
+These commands will ask for your password: type it in.
+
+:warning: When you type your password, nothing will show up on the screen, **that's normal**. This is a security feature to mask not only your password as a whole but also its length. Just type in your password and when you're done, press `ENTER`.
+
+### GitHub CLI
```

Comment:
❓ Should we remove the `gh_cli` partial from the [windows](https://github.com/lewagon/setup/blob/76777ebc348e1d041bfb3796cd69b3a2376abf14/build.rb#L47) / [ubuntu](https://github.com/lewagon/setup/blob/76777ebc348e1d041bfb3796cd69b3a2376abf14/build.rb#L72) builds?

Reply:
No we don't. `gh_cli` partial is about configuring `gh` and is common to the three setups. The installation of `gh` is in the `macos_homebrew` partial for macOS and it the `git` partial for Windows and Linux.